Pipfile

```diff
@@ -27,41 +27,38 @@ pycountry = ">=19.7.15"
 angular-gettext-babel= ">=0.1"
 # TODO: causes datetime serialize error
 celery = "<4.3.0"
-invenio-oaiharvester = {editable = true,ref = "v1.0.0a4",git = "https://github.com/inveniosoftware/invenio-oaiharvester.git"}
-invenio-circulation = {editable = true,git = "https://github.com/inveniosoftware/invenio-circulation.git",ref = "v1.0.0a16"}
-invenio = {version = "~=3.1.0",extras = ["base", "postgresql", "auth", "elasticsearch6" ]}
+invenio-oaiharvester = {editable = true, ref = "v1.0.0a4", git = "https://github.com/inveniosoftware/invenio-oaiharvester.git"}
+invenio-circulation = {editable = true, git = "https://github.com/inveniosoftware/invenio-circulation.git", ref = "v1.0.0a16"}
+invenio = {version = "==3.2.1", extras = ["base", "postgresql", "auth", "elasticsearch6" ]}
 dojson = ">=1.3.2"
-# get rid of invenio-search-ui
-invenio-indexer = ">=1.0.1,<1.1.0"
-invenio-jsonschemas = ">=1.0.0,<1.1.0"
-invenio-oaiserver = ">=1.0.3,<1.1.0"
-invenio-pidstore = ">=1.0.0,<1.1.0"
-invenio-records-rest = ">=1.4.0,<1.5.0"
+# metadata bundle without invenio-search-ui
+invenio-indexer = ">=1.1.1,<1.2.0"
```

Comment:
Can we remove all invenio-xx constraints, except invenio-base?

Reply:
It was also my first goal.
Then I saw RERO don't use invenio-search-ui and don't want to.
The invenio-xx constraints are present to avoid using invenio-search-ui.
So I just keep this as it is (but update dependencies).

Reply:
Can you confirm that all the dependencies constraints are the same as Invenio?

Reply:
I confirm. Related to https://github.com/inveniosoftware/invenio/blob/maint-3.2/setup.py#L50-L57